nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
```

Comment:
I've been mulling over moving it to a separate package.

Concerned that it will end up looking like `nsdq.logf(logger.LOG_WARN, "something")` and a little baffled how to handle the config side. Normally could write a custom marshaling functions to deal with the mapping from string to int but i do not see how to support with go-options.

I am ok with you folks refactoring it later; I am still learning atm.